Fix flickering test and remove the 1s delay from test/scroller.html

limonte opened the pull request:
- bump wct to `^5.0.0`
- fix F-test
- remove unclear test
- remove strange 1s delay after each test

Fixes #322

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/vaadin-date-picker/323)
<!-- Reviewable:end -->

tomivirkki commented:
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[test/scroller.html, line 68 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/323#-Kaf90vzBrNTeAFrkSG2:-Kaf90vzBrNTeAFrkSG3:bmf38dj) ([raw file](https://github.com/vaadin/vaadin-date-picker/blob/b2b541fa931fb6571ea939ad06108a9ba50c0383/test/scroller.html#L68)):*
> ```HTML
>             done();
>           } else {
>             scroller.$.scroller.scrollTop += scroller.itemHeight * 3.7;
> ```

The number was fractional on purpose because the scroller should handle fractional positions. Rounding this to 4 makes the test invalid.

---

*[test/scroller.html, line 90 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/323#-Kaf9q5S2djxkqyEf9Ei:-Kaf9q5S2djxkqyEf9Ej:b2otp50) ([raw file](https://github.com/vaadin/vaadin-date-picker/blob/b2b541fa931fb6571ea939ad06108a9ba50c0383/test/scroller.html#L90)):*
> ```HTML
>       });
> 
>       it('should reflect position as currently visible item index', function() {
> ```

Why is this test removed?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/323)*
<!-- Sent from Reviewable.io -->

limonte commented:
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[test/scroller.html, line 68 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/323#-Kaf90vzBrNTeAFrkSG2:-Kafg4AQRh25lviWnwFS:b-n1za2f) ([raw file](https://github.com/vaadin/vaadin-date-picker/blob/b2b541fa931fb6571ea939ad06108a9ba50c0383/test/scroller.html#L68)):*
<details><summary><i>Previously, tomivirkki (Tomi Virkki) wrote…</i></summary><blockquote>

The number was fractional on purpose because the scroller should handle fractional positions. Rounding this to 4 makes the test invalid.
</blockquote></details>

Reverted, thanks for the explanation!

---

*[test/scroller.html, line 90 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/323#-Kaf9q5S2djxkqyEf9Ei:-Kafg8AcryJWcRy66UWL:b-1iaen3) ([raw file](https://github.com/vaadin/vaadin-date-picker/blob/b2b541fa931fb6571ea939ad06108a9ba50c0383/test/scroller.html#L90)):*
<details><summary><i>Previously, tomivirkki (Tomi Virkki) wrote…</i></summary><blockquote>

Why is this test removed?
</blockquote></details>

My bad, reverted.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/323)*
<!-- Sent from Reviewable.io -->

manolo commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/323#-:-Kagm6IAksXHrUqT6fDO:bnfp4nl)*
<!-- Sent from Reviewable.io -->

tomivirkki commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/323#-:-KagwlyL6TqTlFPyrLgw:bnfp4nl)*
<!-- Sent from Reviewable.io -->
